### ISSUE TITLE
Bug: Error Not Visible on UI When No GitOps Helm Deployment Fails Due to Values Schema Error.

### ISSUE THREAD

**Issue by YashasviDevtron** [devtron-labs/devtron#5546]
### 📜 Description

When upgrading a Helm chart,  error of values shema is not getting displayed on  on the UI leading to deployment failures without proper notification.

### Affected areas

Other CRITICAL functionality

### Additional affected areas

None

### Prod/Non-prod environments?

Prod

### Is User unblocked?

No

### How was the user un-blocked?

TEMPORARILY - By doing some changes from the backend/DB

### Impact on Enterprise

Deployment failures due to errors in the values.yaml file are not getting displayed on UI  during helm chart deployment.

### 👟 Steps to replicate the Issue

1. Deploy a helm chart.
2. Modify the values.yaml file with incorrect indentation or a schema error.
3. Then update and deploy the chart again.

### 👍 Expected behavior

The UI should display an error message when a values schema error occurs when the helm chart is getting updated and deployed.

### 👎 Actual Behavior

The UI allows the Helm chart deployment to proceed without notifying the user of any errors in the values schema.

### ☸ Kubernetes version

1.3

### Cloud provider

<details>
aws
</details>

### 🌍 Browser

Chrome

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find any similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/devtron-labs/devtron/blob/main/CODE_OF_CONDUCT.md)

AB#10210

**Comment by prakarsh-dt** [comment]
Duplicate of #5354
